# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/atk/AccessibleWrap.h
# Commit: e807c6507634
# Full Hash: e807c6507634bdeca647372d8815570d7750b3b2
# Author: James Teh <jteh@mozilla.com>
# Date: 2022-04-02 09:44:13
# Regressor Bug: 1735970
# File Overlap Count: 1
# Description:
#   Bug 1735970 part 15: Support TableAccessibleBase and TableCellAccessibleBase for ATK. r=morgan
#   
#   We can use the base classes for both local and cached remote Accessibles.
#   However, non-cached RemoteAccessibles need to be handled separately still because we can't implement the table interfaces for those.
#   
# ==============================================================================

diff -r 03e98595f217 -r e807c6507634 accessible/atk/AccessibleWrap.h
--- a/accessible/atk/AccessibleWrap.h	Fri Apr 01 09:50:01 2022 +0000
+++ b/accessible/atk/AccessibleWrap.h	Fri Apr 01 09:50:01 2022 +0000
@@ -70,10 +70,10 @@
 
   static void GetKeyBinding(LocalAccessible* aAccessible, nsAString& aResult);
 
-  static LocalAccessible* GetColumnHeader(TableAccessible* aAccessible,
-                                          int32_t aColIdx);
-  static LocalAccessible* GetRowHeader(TableAccessible* aAccessible,
-                                       int32_t aRowIdx);
+  static Accessible* GetColumnHeader(TableAccessibleBase* aAccessible,
+                                     int32_t aColIdx);
+  static Accessible* GetRowHeader(TableAccessibleBase* aAccessible,
+                                  int32_t aRowIdx);
 
  protected:
   nsresult FireAtkStateChangeEvent(AccEvent* aEvent, AtkObject* aObject);